files/ko/web/api/audiocontext/index.html

```diff
@@ -5,92 +5,60 @@
   - 오디오
```

Comment:
```suggestion
  - API
  - Audio
  - AudioContext
  - Interface
  - Reference
  - Web Audio API
  - sound
```

태그들도 동기화 해주시면 좋을거 같아요!